### PR TITLE
Fixed the Exception handling

### PR DESCRIPTION
**Pull request by MikaelMani99**
Fixed the Exception handling, and added a more standard way or returning the exception.

### CODE REVIEW

**Comment by MaggaSK97** [comment]
Fix warning in commit: Fixed the Exception handling

File: Web/utkeyrslukerfi-web/src/styles/main.css [hunk 1]
```diff
@@ -5,3 +5,7 @@
 .navigation-link {
   text-decoration: none;
 }
+
+.hidden {
+  display: none;
```
Comment (ertolv) [inline Web/utkeyrslukerfi-web/src/styles/main.css:10]: just use d-none boostrap class

File: Web/utkeyrslukerfi-web/src/styles/main.css [hunk 2]
```diff
@@ -5,3 +5,7 @@
 .navigation-link {
   text-decoration: none;
```
Comment (ertolv) [inline Web/utkeyrslukerfi-web/src/styles/main.css:6]: just use text-decoration-none from bootstrap

**Review by ertolv — CHANGES_REQUESTED** [review]
Can you use bootstrap classess instead of writing proprietary css

**Review by ertolv — APPROVED** [review]

**Pull request merged**